test/runtests.jl

```diff
@@ -85,7 +85,11 @@ rectangularQ(Q::LinearAlgebra.AbstractQ) = convert(Array, Q)
                     sq = size(q.factors, 2)
                     @test *(Matrix{eltyb}(I, sq, sq), adjoint(q)) * squareQ(q) ≈ Matrix(I, sq, sq) atol=5000ε
                     if eltya != Int
-                        @test Matrix{eltyb}(I, a_1, a_1)*q ≈ convert(AbstractMatrix{tab}, q)
+                        if VERSION < v"1.10-"
+                            @test Matrix{eltyb}(I, a_1, a_1)*q ≈ convert(AbstractMatrix{tab}, q)
+                        else
+                            @test Matrix{eltyb}(I, a_1, a_1)*q ≈ squareQ(convert(LinearAlgebra.AbstractQ{tab}, q))
+                        end
```

Comment:
I couldn't find a version-independent form of this test that would pass on all versions. There is an accuracy issue hidden here.